packages/office-ui-fabric-react/src/components/Callout/Callout.Props.ts

```diff
@@ -106,6 +106,11 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
   setInitialFocus?: boolean;
 
   /**
+   * Set the max height of Callout component.
+   */
+  maxHeight?: number;
```

Comment:
I remember trying this change on local box but it had some scroll width issues. Can you check if scroll is included in the popup width? Also this would fix #954 as well. Thanks